<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Data Base</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="OpenDataBase">
           <property name="text">
            <string>üìÇOpen Data Base</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeDataBase">
           <property name="text">
            <string>‚úñClose Data Base</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Row operations</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="renamTable">
           <property name="text">
            <string>‚úé Rename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="create">
           <property name="text">
            <string>Ôºã Create</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dropTable">
           <property name="text">
            <string>‚úñ Delete Table</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Selector</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="show_table">
           <property name="text">
            <string>‚ü≤ Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>show RoID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="select_table"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Edit Table</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="insert_table">
           <property name="text">
            <string>‚Üß Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="edit_cell">
           <property name="text">
            <string>‚úé Edit Cell</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove">
           <property name="text">
            <string>üóë Remove </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Search</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Find where : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ColName"/>
         </item>
         <item>
          <widget class="QComboBox" name="operation_tags"/>
         </item>
         <item>
          <widget class="QLineEdit" name="key"/>
         </item>
         <item>
          <widget class="QPushButton" name="searchBox">
           <property name="text">
            <string>üîç Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>::</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSlider" name="zoomSlider">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="index">
        <property name="text">
         <string>No index selected</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="zoomPercent">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="tableName">
        <property name="text">
         <string>No table is open</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="dataBaseName">
        <property name="text">
         <string>No database is open</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>::</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionGet_backup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUIFont"/>
    <addaction name="actionTable_font_edit"/>
    <addaction name="separator"/>
    <addaction name="actionreset_UI_font"/>
    <addaction name="actionReset_Table_font"/>
    <addaction name="separator"/>
    <addaction name="actionclear_all"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionnew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUIFont">
   <property name="text">
    <string>UI font edit</string>
   </property>
  </action>
  <action name="actionclear_all">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>clear all</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShow_table">
   <property name="text">
    <string>Show table</string>
   </property>
  </action>
  <action name="actionShow_ROWID">
   <property name="text">
    <string>‚úì Show ROWID</string>
   </property>
  </action>
  <action name="actionTable_font_edit">
   <property name="text">
    <string>Table font edit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionGet_backup">
   <property name="text">
    <string>Get backup</string>
   </property>
  </action>
  <action name="actionreset_UI_font">
   <property name="text">
    <string>reset UI font</string>
   </property>
  </action>
  <action name="actionReset_Table_font">
   <property name="text">
    <string>Reset Table font</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
